src/main/resources/hudson/tasks/junit/JUnitResultArchiver/help-keepLongStdio.html

```diff
@@ -1,4 +1,5 @@
 <div>
+   (Deprecated; use stdioRetention instead)
```

Comment:
probably something like this to emphasise it:
```suggestion
  <strong>Deprecated, use stdioRetention instead.</strong>
  <p>
```